data.sql

```diff
@@ -1,7 +1,67 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Agumon',02/03/2020,0,'true',10.23);
-INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Gabumon',11/15/2018,2,'true',8);
-INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Pikachu',01/07/2021,1,'false',15.04);
-INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Devimon',05/12/2017,5,'true',11);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Agumon','2020-02-03',0,'true',10.23);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Gabumon','2018-11-15',2,'true',8);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Pikachu','2021-01-07',1,'false',15.04);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Devimon','2017-05-12',5,'true',11);
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Charmander','2020-02-08',0,'false',-11);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Plantmon','2021-11-15',2,'true',-5.7);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Squirtle','1993-04-02',3,'true',-12.13);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Angemon','2005-06-12',1,'true',-45);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Boarmon','2005-06-07',7,'true',20.4);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Blossom','1998-10-13',3,'true',17);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Ditto','2022-05-14',4,'true',22);
+
+UPDATE animals SET species = 'unspecified';
+
+BEGIN;
+
+UPDATE animals 
+SET species = 'unspecified';
+
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/165406787-faf5809d-1003-466c-9588-c394ba90b2e3.png)

- [ ] As your column `ID` in the animal's table is not auto-generated and should not be null (`NOT NULL`), you need to insert it also. or autogenerate your id in the animal table. Example on how to auto-generate an ID in Postgres :point_down: 

```sql
-- Note that this is just an example, you can solve the error above the way you can 
CREATE TABLE animals(                                                                                                     
   id INT GENERATED ALWAYS AS IDENTITY,
   name           TEXT    NOT NULL,
   date_of_birth            DATE     NOT NULL,
   escape_attempts      INT NOT NULL,
   neutered        BOOLEAN NOT NULL,
   weight_kg         REAL
);
```

Reply:
same here